akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/EventSourcedBehavior.scala

```diff
@@ -63,7 +63,7 @@ object EventSourcedBehavior {
       emptyState: State,
       commandHandler: (State, Command) => ReplyEffect[Event, State],
       eventHandler: (State, Event) => State): EventSourcedBehavior[Command, Event, State] = {
-    val loggerClass = LoggerClass.detectLoggerClassFromStack(classOf[EventSourcedBehavior[_, _, _]])
+    val loggerClass = LoggerClass.detectLoggerClassFromStack(classOf[EventSourcedBehavior[_, _, _]], logPrefixSkipList)
```

Comment:
I am just wondering if there was a reason it was omitted here.

Reply:
I think the reason could be that `withEnforcedReplies` calls the `apply`. @johanandren might be able to confirm.

Reply:
I'm afraid I don't remember, the tests should catch if it breaks something?